src/terra-core/TerraCoreDataModel.ttl

```diff
@@ -3260,11 +3181,11 @@ TerraDCAT_ap:hasOwner
 .
 TerraDCAT_ap:hasPrincipalInvestigator
   rdfs:range [
-    owl:unionOf (
-      xsd:string
-      TerraCore:PrincipalInvestigator 
-    );
-  ];
+      owl:unionOf (
+          xsd:string
```

Comment:
can't unionOf datatype and object

Reply:
Deferred as there are many of these - See[ DM-507](https://broadworkbench.atlassian.net/browse/DM-507)